src/pages/offer/PeersTable.tsx

```diff
@@ -61,18 +69,26 @@ const PeerRow: React.FC<PeerRowProps> = ({ index, peer }) => {
             <Cell>
               <A href={`/peer/${peer.id}`}>{peer.id}</A>
             </Cell>
+            {/* Current CC */}
+            <Cell>
+              {peer.currentCapacityCommitment ? (
+                <A href={`/capacity/${peer.currentCapacityCommitment?.id}`}>
+                  {formatHexData(peer.currentCapacityCommitment?.id, 15, 15)}
+                </A>
+              ) : (
+                '-'
+              )}
+            </Cell>
             {/* Compute units */}
             <Cell>
               <Text size={12}>{peer.computeUnits.length}</Text>
             </Cell>
-            {/* Current Capacity Commitment */}
+            {/* Status */}
```

Comment:
I would just remove this. Serves no purpose
```suggestion
```

Reply:
I agree that such comments are likely redundant and can be removed throughout the codebase